__tests__/app.test.js

```diff
@@ -61,6 +62,27 @@ describe('Reviews  get request test block', () => {
             })
         })
     })
+    test.only("/api/reviews/3?count=comment returns an reviews object with new property including total amount of comments with that id", () => {
+        return request(app).get("/api/reviews/1?count=comment").expect(200).then((res) => {
```

Comment:
Not quite sure why you've done it as a query. This is a refactor of a route you already have, not a query on the route. 
You need to change the model for the /api/reviews/:review_id route